BrightspaceHypermediaComponents/foundation-components
# feat: visibility toggle added

Context:
[US114990](https://rally1.rallydev.com/#/357252966636d/custom/367300408400?detail=%2Fuserstory%2F378717803400)

Implementing a component for a visibility toggle and making the learning path editor use it.

Quality:
Tested in polarisdev manually.
Tests:
- load learning path that is visible, toggle to be hidden, press save and close, reloading page learning path is hidden
- load learning path that is hidden, toggle to be visible, press save and close, reloading page learning path is visible
- toggling twice, then pressing save and close, causes learning path visibility to be unchanged
- toggling multiple times and pressing save and close, results in updated visibility
- toggling multiple times and pressing cancel, results in visibility being unchanged
Toggling then canceling does not revert the toggle currently. Since learning paths with close on cancel, this should not be necessary but should be implemented.
